src/calliope_pathways/models/italy_stationary/model_config/techs.yaml

```diff
@@ -0,0 +1,305 @@
+
+tech_groups:
+  cost_dim_setter:
+    cost_flow_cap:
+      data: null
+      index: monetary
+      dims: costs
+    cost_om_annual:
+      data: null
+      index: monetary
+      dims: costs
+    cost_flow_in:
+      data: null
+      index: monetary
+      dims: costs
+    cost_flow_out:
+      data: null
+      index: monetary
+      dims: costs
+    cost_storage_cap:
+      data: null
+      index: monetary
+      dims: costs
+    cost_area_use:
+      data: null
+      index: monetary
+      dims: costs
+    cost_source_cap:
+      data: null
+      index: monetary
+      dims: costs
+    cost_interest_rate:
+      data: 0.10
+      index: monetary
+      dims: costs
+
+  inter_zonal_transmission:
+    name: "Transmission intra-zonal electricity"
+    color: '#966F9E'
+    carrier_in: power
+    carrier_out: power
+    base_tech: transmission
+    lifetime: 40
+    flow_out_eff: 0.96
+    cost_interest_rate:
+      data: 0.10
+      index: monetary
+      dims: costs
+    cost_flow_out:
+      data: 0.0011  # EUR2010/kWh, from Lombardi
+      index: monetary
+      dims: costs
+    cost_flow_cap:
+      data: 450   # EUR2010/kW, from Lombardi
+      index: monetary
+      dims: costs
+
+techs: # Color scheme is Tol Qual Muted + some extras
+  ##
+  # Supply data, all values from Lombardi's paper.
+  ##
+  geothermal:
+    name: "Traditional geothermal energy"
+    color: '#873737'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    lifetime: 50
+
+    cost_flow_cap.data: 4062 # EUR2010/kW lower than average IRENA cost in 2022USD normalized to EUR2010 for Italy.
+    cost_om_annual.data: 77 # EUR2010/kW/yr
+
+  hydropower: # merged reservoir+basin and RoR, average values
+    name: "Hydropower"
+    color: '#64D7CE'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    include_storage: true  # TODO: unclear how to make this work
+    source_unit: per_cap
+    source_eff: 0.635
+    lifetime: 55
+
+    # from RSE, averaged 2015 estimates (2020 for RoR)
+    cost_flow_cap.data: 4022 # EUR2010/kW
+    cost_om_annual.data: 73 # EUR2010/kW/yr
+
+  waste:
+    name: "Waste to energy plant"
+    color: '#6A0241'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    flow_out_eff: 0.28
+    lifetime: 20
+
+    # from RSE, 2015 estimates for electric-only
+    cost_flow_cap.data: 3743 # EUR2010/kW
+    cost_om_annual.data: 348 # EUR2010/kW/yr
+    # from Lombardi
+    cost_flow_in.data: -0.1305 # EUR2010/kWh
+    cost_flow_out.data: 0.03305 # EUR2010/kWh
+
+  bioenergy:
+    name: "Biogas, biofuel and biomass power generation"
+    color: '#5AA24D'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    flow_out_eff: 0.29
+    lifetime: 18 # average in Lombardi's study
+
+    # from RSE, averaged solid, gas and liquid
+    cost_flow_cap.data: 4137 # EUR2010/kW
```

Comment:
why not put these into a CSV file since you're parsing the data anyway? Saves you having to have the `cost_dim_setter`

Reply:
I thought about it, but I went against it since this is a stationary model and this is stationary data.
Once I add non-stationary scenarios with varying prices, I'll move this to a separate file.
I'll keep it as-is, for now.